WBooks/BookDetails/BookDetailsViewController.swift

```diff
@@ -81,9 +88,34 @@ class BookDetailsViewController: UIViewController {
         }
     }
     
+    private func loadComment() {
+        bookDetailsViewModel.getComments { [weak self] in
+            self?.bookDetailsView.commentsTableView.reloadData()
+        }
+    }
+    
     func showAlertBookUnavailble(title: String, message: String) {
         let alert = UIAlertController(title: title, message: message, preferredStyle: UIAlertController.Style.alert)
         alert.addAction(UIAlertAction(title: "Ok", style: UIAlertAction.Style.default, handler: nil))
         self.present(alert, animated: true, completion: nil)
     }
 }
+
+extension BookDetailsViewController: UITableViewDelegate, UITableViewDataSource {
+    // MARK: - Table view data source
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return bookDetailsViewModel.comments.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath)
+                as? BookCommentsCell else { return UITableViewCell() }
+        let comment = bookDetailsViewModel.comments[indexPath.row]
+        bookDetailsViewModel.getUser(id: comment.userID) { [weak self] in
+            cell.usernameLabel.text = self?.bookDetailsViewModel.user?.username
+        }
+        cell.userImage.image = UIImage(named: "img_user1")
```

Comment:
if using a static image, you can create a UIImage extension and add it there to have them centralized